### DISCUSSION TITLE
Metadata panel size

### DISCUSSION THREAD

**Discussion by TB-HunBoka** [dbeaver/dbeaver#34173] [Q&A] [answered]
Hello Everyone,

I updated the dbeaver version to the Version 24.0.5.202405191503, and now my metadata panel has a very small text:
![image](https://github.com/dbeaver/dbeaver/assets/126617861/a3ccfd35-cfa3-4f98-9fb8-5fe3f85d156e)
Anyone had the same issue or any idea to fix it?

Thanks,

**Comment by TB-HunBoka** [comment]
It was some kind of conflict with the darkest dark theme. I was unable to use the export function and the Key-s.
![image](https://github.com/dbeaver/dbeaver/assets/126617861/18fc79c2-f97c-44fc-b572-2927dcefca61)

An error has occurred. See error log for more details.
Cannot invoke "com.codeaffine.eclipse.swt.widget.scrollable.context.Reconciliation.runWithSuspendedBoundsReconciliation(java.lang.Runnable)" because "this.reconciliation" is null

 Disabling the Darkest Dark Theme, fixed the problem.

**Reply by E1izabeth** [reply]
Could you provide full record of this error or whole log files? 
You can find instructions about the logs location here https://dbeaver.com/docs/wiki/Log-files/

**Reply by TB-HunBoka** [reply]
> Could you provide full record of this error or whole log files? You can find instructions about the logs location here https://dbeaver.com/docs/wiki/Log-files/

I experienced problems with scrollable windows all over the app, therefore here are 2 logs generated by 1. file / export and 2. user interface / Keys 
[eclips scrollable error at export window.log](https://github.com/dbeaver/dbeaver/files/15465605/eclips.scrollable.error.at.export.window.log)
[eclips scrollable error at Keys settings.log](https://github.com/dbeaver/dbeaver/files/15465606/eclips.scrollable.error.at.Keys.settings.log)

Also, the error logs are invisible in darkest dark's light and in darkest dark's dark mode too, so it could not load it correctly.
![image](https://github.com/dbeaver/dbeaver/assets/126617861/e7a76e50-940c-4739-ab24-52fde8e979da)

Do you want me to try it with a clean install? I was using the built in version upgrade, but not checked in the clean install checkbox.

**Reply by LonwoLonwo** [reply]
Maybe a clean install will help, yes.

**Reply by TB-HunBoka** [reply]
> Maybe a clean install will help, yes.

I tried a clean install, but I experience the same issue after installing the darkest dark theme.
I tried the early access version too, same problem.
After Installing the 24.0.4.202405051837 version, I don't experience this issue anymore.

Are you not experiencing the same problem?

**Reply by TB-HunBoka** [reply]
What I can add to this is that the darkest dark settings sometimes changes the error behavior.
Change the icon color setting generates Unable to generate image error
Changing the workbenche theme to Darkest Custom generates more error.
Also I tried it on another machine and experienced the same problem.
Example setup:
![kép](https://github.com/dbeaver/dbeaver/assets/126617861/96add005-86c3-4a5a-8b51-c062d2f71b9b)

Shall I open a bug ticket? Can you please confirm that you can reproduce the issue?

**Reply by E1izabeth** [reply, marked as answer]
Better opening an issue, yes. Thank you

**Reply by rsimmermon** [reply]
I can confirm that I, too, have this same exact problem.
  DevStyle (includes Darkest Dark Theme)	2024.1.0.202402281519	com.genuitec.eclipse.theming.core.feature.feature.group	CodeTogether Inc.
DBeaver Community - Version 24.0.5.202405191503

**Reply by E1izabeth** [reply]
I created an issue for your case https://github.com/dbeaver/dbeaver/issues/34210

**Reply by TB-HunBoka** [reply]
I also created one https://github.com/dbeaver/dbeaver/issues/34209

**Reply by E1izabeth** [reply]
Thank you. I'll close mine then
